admin/customers.php

```diff
@@ -91,7 +91,7 @@
                 $customers_fax = zen_db_prepare_input($_POST['customers_fax']);
             }
             $customers_newsletter = zen_db_prepare_input($_POST['customers_newsletter']);
-            $customers_group_pricing = (int)$_POST['customers_group_pricing'];
+            $customers_group_pricing = (int)$_POST['customers_group_pricing'] ?? 0; //- No present if wholesale pricing is selected!
```

Comment:
'No' vs 'Not' typo in comment.

Reply:
Yep, and casting has a higher priority than null-coalesce (sheesh).